app/main.go

```diff
@@ -118,6 +118,8 @@ type options struct {
 	Dry   bool `long:"dry" env:"DRY" description:"dry mode, no bans"`
 	Dbg   bool `long:"dbg" env:"DEBUG" description:"debug mode"`
 	TGDbg bool `long:"tg-dbg" env:"TG_DEBUG" description:"telegram debug mode"`
+
+	LoggingFormat string `long:"logging-format" env:"LOGGING_FORMAT" default:"text" description:"logging format, 'json' or 'text'"`
```

Comment:
This should be added to the README, probably to the list of parameters. It may also be worth mentioning in the detailed explanation of the parameters as well.

Reply:
fixed, but my formatter did some changes in the readme, I hope don't mind, they do not look destructive to me